sympy/calculus/tests/test_util.py

```diff
@@ -152,6 +152,7 @@ def test_periodicity():
     assert periodicity(log(x), x) is None
     assert periodicity(exp(x)**sin(x), x) is None
     assert periodicity(sin(x)**y, y) is None
+    assert periodicity(exp(asin(x)), x) is None
```

Comment:
Hi @kc611 

Could you also add a test for the previous #12620 ?

```python3
assert periodicity(Abs(x**2 - 3*x),x) is None
```

Thank you

Reply:
Sure